acceptance/tests/invalid_ssl_config.rb

```diff
@@ -4,10 +4,31 @@
 
 agent1 = agents[0]
 
+# @param host the host to check the logile on
+# @param logfile path to log file
+# @param expected the string or pattern expected in the log
+def wait_for_log_entry(host, logfile, expected)
+  # If the expected log entry does not appear in log within 30 seconds, then do an
+  # explicit assertion so we get a test fail (not an unhandled error),
+  # and see the log contents in the test failure output
+  begin
+    retry_on(host, "grep '#{expected}' #{logfile}", {:max_retries => 30,
+                                                     :retry_interval => 1})
+  rescue
```

Comment:
I think a different helper file, as the config_helper was meant to be just helping with pxp-agent.conf

I can do that... but actually I think this could go into beaker itself, I can look into doing that